great_expectations/rule_based_profiler/data_assistant/data_assistant_runner.py

```diff
@@ -197,7 +197,7 @@ def run(
             self._get_method_signature_parameters_for_variables_directives()
         )
 
-        func_sig = Signature(
+        func_sig: Signature = Signature(
```

Comment:
This is fine but the LHS is a `Signature` constructor so I don't think the type declaration adds to the understanding of the code here. I would remove.

Reply:
@billdirks I wanted to add it back, but will un-add, because it is indeed clear enough here.  Thanks.